### PR TITLE
API docs migration  4

### PR DESCRIPTION
**Pull request by AaronForinton**
todo: resolve internal links

https://web.dev/reporting-api -> https://developer.chrome.com/articles/reporting-api/
https://web.dev/reporting-api-migration/ -> https://developer.chrome.com/blog/reporting-api-migration/
https://web.dev/urlpattern/ -> https://developer.chrome.com/articles/urlpattern
https://web.dev/pwa-url-handler/ -> https://developer.chrome.com/articles/pwa-url-handler
https://web.dev/mediastreamtrack-insertable-media-processing/ -> https://developer.chrome.com/articles/mediastreamtrack-insertable-media-processing/

### CODE REVIEW

File: site/en/articles/mediastreamtrack-insertable-media-processing/index.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+[Streams API](https://streams.spec.whatwg.org/)). These streams can be manipulated to introduce new
+components.
+
+Granting developers access to the video (or audio) stream directly allows them to apply
+modifications directly to the stream. In contrast, realizing the same video manipulation task with
+traditional methods requires developers to use intermediaries such as `<canvas>` elements. (For
+details of this type of process, see, for example,
+[video + canvas = magic](https://html5doctor.com/video-canvas-magic/).)
+
+## Use cases
```
Comment (rachelandrew) [inline site/en/articles/mediastreamtrack-insertable-media-processing/index.md:43]:
```suggestion
## Browser support 

Insertable streams for `MediaStreamTrack` is supported from Chrome 94.

## Use cases
```

File: site/en/articles/reporting-api-migration/index.md [hunk 2]
```diff
@@ -0,0 +1,483 @@
+---
+layout: 'layouts/blog-post.njk'
```
Comment (rachelandrew) [inline site/en/articles/reporting-api-migration/index.md:2]: Can we move this one to the blog? It's essentially news. So blog/reporting-api-migration/

File: site/en/articles/reporting-api-migration/index.md [hunk 3]
```diff
@@ -0,0 +1,483 @@
...
+hero: image/O2RNUyVSLubjvENAT3e7JSdqSOx1/PYEe5UP3bVYzPMXdQc0X.jpg
+alt: A person and their refection on wet sand, symbolizing the migration from the legacy to the new Reporting API.
+tags:
+  - monitoring
+  - security
+---
+
+{% Aside 'caution' %} This post covers how to migrate to the new version of the Reporting
+API (v1). If you're new to the Reporting API or need detailed usage examples, head over to the [main
+post on the Reporting API](/reporting-api) instead. {% endAside %}
```
Comment (rachelandrew) [inline site/en/articles/reporting-api-migration/index.md:19]: URL needs updating, and check for other mentions.

File: site/en/articles/urlpattern/index.md [hunk 4]
```diff
@@ -0,0 +1,412 @@
...
+response to network requests. The concepts in this example could be applied to
+other routing scenarios as well, either server-side or client-side.
+
+{% Glitch {
+id: 'urlpattern-sw-routing',
+path: 'service-worker.js',
+previewSize: 35
+} %}
+
+## Browser support and polyfills
```
Comment (rachelandrew) [inline site/en/articles/urlpattern/index.md:354]: Can we move this support bit to the top somewhere, just under the initial description.

**Review by rachelandrew — COMMENTED** [review]
Couple of changes here.

**Comment by AaronForinton** [comment]
> Couple of changes here.

I've applied those

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**